core/src/main/java/ai/timefold/solver/core/impl/domain/solution/descriptor/DefaultPlanningListVariableMetaModel.java

```diff
@@ -30,6 +32,22 @@ public boolean allowsUnassignedValues() {
         return false;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        // DO NOT USE entity in equality checks;
```

Comment:
```suggestion
        // Do not use entity in equality checks;
```

Let's not shout at people.